editor.planx.uk/src/routes/index.tsx

```diff
@@ -30,6 +31,13 @@ const editorRoutes = mount({
         }),
   ),
 
+  "/:team/:flow/:sessionId/verify-email": map((req) => {
+    return route({
+      title: makeTitle("Verify your email"),
+      view: <VerifyEmail params={req.params} />,
+    });
+  }),
```

Comment:
nit: for consistency, I wonder if this route name could be `/download-application` (then download link remains `/download-application-files`) ?